rpcs3/Emu/Cell/RawSPUThread.cpp

```diff
@@ -392,14 +392,35 @@ void spu_load_exec(const spu_exec_object& elf)
 
 	const auto funcs = spu->discover_functions(0, { spu->ls , SPU_LS_SIZE }, true, umax);
 
-	for (u32 addr : funcs)
+	if (spu_log.notice && !funcs.empty())
 	{
-		spu_log.success("Found SPU function at: 0x%08x", addr);
+		std::string to_log;
+
+		for (usz i = 0; i < funcs.size(); i++)
+		{
+			if (i == 0 && funcs.size() < 4)
```

Comment:
why 4 though? this is barely any improvement

Reply:
I tried 8 but because this is not monospace text the differences in space accumulated each linegreater than 4 and it looked bad.